purchase_request_operating_unit/wizard/purchase_request_line_make_purchase_order.py

```diff
@@ -21,10 +21,18 @@ class PurchaseRequestLineMakePurchaseOrder(models.TransientModel):
     def default_get(self, fields):
         res = super(PurchaseRequestLineMakePurchaseOrder, self).\
             default_get(fields)
-        request_line_obj = self.env['purchase.request.line']
-        request_line_ids = self._context.get('active_ids', [])
+        active_model = self.env.context.get('active_model', False)
+        request_line_ids = []
```

Comment:
I think it is not necessary to predefine this variable as long there is no lop, you can define the variable inside de if statament. don't you?

Reply:
Yes I tought so too, but I opted to stay on the same logic of the original PR for the purchase_request module: https://github.com/OCA/purchase-workflow/commit/4fe3697#diff-e458197a92be2847dc330973def5bc39R99

Though I do think it is unnecessary in both cases.

Reply:
I see it was there before. But it will be an improvement,  you could take advantage of this PR and change it :) Not blocking anyway

Reply:
Upon further investigation I think it was done to simplify the exit case when no request lines are defined OR when there is a bad context here: https://github.com/OCA/purchase-workflow/commit/4fe3697#diff-e458197a92be2847dc330973def5bc39R106

Otherwise we would have to individually test if there are some request lines or if the context is properly configured. Or wrap the whole logic after on an `if request_line_ids` statement, which will further make the code complex